tests/functional/test_credentials.py

```diff
@@ -198,7 +198,7 @@ def create_assume_role_response(self, credentials, expiration=None):
             },
             'AssumedRoleUser': {
                 'AssumedRoleId': 'myroleid',
-                'Arn': 'arn:aws:iam::1234567890:user/myuser',
+                'Arn': f'arn:aws:iam::{credentials.account_id}:user/myuser',
```

Comment:
Since the `AssumedRoleUser` object with a valid ARN was already a part of this test helper function, the `account_id` in the resulting credentials object will always resolve to a non-None value. So this needed to be updated. Another option would be to delete `AssumedRoleUser`, which would let us remove the below account ID line.